benchmark/NClient.Benchmark.Client/JsonHttpResponseClient/JsonHttpResponseClientBenchmark.cs

```diff
@@ -0,0 +1,161 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text;
+using System.Text.Json;
+using BenchmarkDotNet.Attributes;
+using Flurl.Http;
+using RestSharp;
+using WireMock.Server;
+
+// ReSharper disable RedundantNameQualifier
+namespace NClient.Benchmark.Client.JsonHttpResponseClient
+{
+    [SimpleJob(invocationCount: 2000, targetCount: 10)]
+    public class JsonHttpResponseClientBenchmark
+    {
+        private static readonly string[] Ids = { "id-1", "id-2", "id-3", "id-4" };
+        
+        private IWireMockServer _api = null!;
+        
+        private HttpClient _httpClient = null!;
+        private RestSharp.RestClient _restSharpClient = null!;
+        private FlurlClient _flurlClient = null!;
+        private INClientJsonHttpResponseClient _nclient = null!;
+        private IRefitJsonHttpResponseClient _refitClient = null!;
+        private IRestEaseJsonHttpResponseClient _restEaseClient = null!;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _api = JsonHttpResponseApiMock.MockMethod(Ids);
+            
+            _httpClient = new HttpClient();
+            _restSharpClient = new RestSharp.RestClient(_api.Urls.First());
+            _flurlClient = new FlurlClient(_api.Urls.First());
+            _nclient = NClientGallery.Clients.GetRest().For<INClientJsonHttpResponseClient>(_api.Urls.First()).Build();
+            _refitClient = Refit.RestService.For<IRefitJsonHttpResponseClient>(_api.Urls.First());
+            _restEaseClient = RestEase.RestClient.For<IRestEaseJsonHttpResponseClient>(_api.Urls.First());
+        }
+        
+        [Benchmark]
```

Comment:
What about "cold start"? Does this library take this into account by default?

Reply:
Yes, warming up by default. Cold start must be specified explicitly: https://benchmarkdotnet.org/articles/guides/choosing-run-strategy.html